### DISCUSSION TITLE
Does this library work with Android?

### DISCUSSION THREAD

**Discussion by TheM1Stery** [amwx/FluentAvalonia#214] [Q&A] [answered]
I want to create a app in Avalonia that works on desktop and android. Will the controls in this library work on Android?

**Comment by TheM1Stery** [comment]
I think it works fine after i tested it. But i only tested NavigationView, haven't tested other controls yet.

**Comment by amwx** [comment, marked as answer]
99% of this library should work with the mobile backends. The controls themselves aren't using any platform specific logic iirc, and even if they did, they'd be checked to prevent crashing. However, give it a test and if you find issues raise a bug report. Though I don't have the mobile workloads installed and don't really know much about mobile development

The only things that will not work with mobile, but are minimal impact:
1- `CoreWindow` because it's a Window
2- Automatic theme resolving in FluentAvaloniaTheme

**Reply by TheM1Stery** [reply]
Thanks for the answer. I'm not an expert in mobile development either, in fact this is my first experience with it. If i ever see a bug, i'll report it. Thanks for your amazing library!
